packages/manager/modules/vps/src/dashboard/vps-dashboard.html

```diff
@@ -557,103 +558,20 @@
         </div>
 
         <div class="col-xm-12 col-md-4">
-            <oui-tile
-                data-heading="{{ ::'vps_tile_subscription' | translate }}"
-                class="h-100"
+            <billing-subscription-tile
+                on-error="$ctrl.onBillingInformationError(error)"
+                service-path="{{ '/vps/' + $ctrl.serviceName }}"
+                with-engagement="$ctrl.isCommitmentAvailable"
+                go-to-commit="$ctrl.goToCommit()"
+                go-to-cancel-commit="$ctrl.goToCancelCommit()"
+                go-to-cancel-resiliation="$ctrl.goToCancelResiliation()"
+                go-to-resiliation="$ctrl.goToResiliation()"
+                service-infos="$ctrl.serviceInfo"
+                highlight-engagement="$ctrl.shouldReengage"
+                tracking-prefix="vps::detail::dashboard"
```

Comment:
it's already defined in packages/manager/modules/vps/src/dashboard/commitment/vps-commitment.routing.js. You can use it

Reply:
This is moved to the dashboard routing resolve